internal/tls/statechannels.org.pem

```diff
@@ -0,0 +1,26 @@
+-----BEGIN CERTIFICATE-----
+MIIEajCCAtKgAwIBAgIRALxmKKJiHaf4cgZxwFTXITEwDQYJKoZIhvcNAQELBQAw
+fzEeMBwGA1UEChMVbWtjZXJ0IGRldmVsb3BtZW50IENBMSowKAYDVQQLDCFtaXNo
```

Comment:
This isn't our real statechannel key is it?

Reply:
Ah nevermind, I see above that it's not.

Reply:
This is a private key used for a self signed certificate. Self signed certificates should not be used in production. The key is checked in for testing ease.